packages/cozy-flags/src/flag.js

```diff
@@ -63,7 +63,7 @@ export const enable = flagsToEnable => {
 export const initializeFromRemote = async client => {
   const {
     data: { attributes }
-  } = await client.stackClient.fetchJSON('/settings/flags')
+  } = await client.stackClient.fetchJSON('GET', '/settings/flags')
```

Comment:
Can't we use client.get('io.cozy.settings', 'flags')?

Reply:
We could, is it blocking for you ?